syntax = "proto3";

import "super.proto";

service ShowTime {
    rpc GetMovieByDate(MovieDate) returns (MovieID) {}
    rpc GetSchedule(Empty) returns (Schedule) {}
}

message Schedule {
    repeated TimeShow schedule = 1;
}

message MovieDate {
    string date = 1;
}

message MovieID {
    repeated string movies = 1;
}